src/main/java/com/avispl/symphony/dal/communicator/nec/multisync/SocketCommunicator.java

```diff
@@ -302,7 +305,7 @@ public static String getHexByteString(CharSequence prefix, CharSequence separato
             this.status.setConnectionState(ConnectionState.Unknown);
             this.destroyChannel();
             if (retryOnError) {
-                return this.send(data, true);
```

Comment:
this will make it retry once when the error occur, not every time timeout occurs